pkgs/development/python-modules/fpdf2/default.nix

```
@@ -0,0 +1,63 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+
+, defusedxml
+, pillow
+, fonttools
+
+, pytestCheckHook
+, qrcode
+, camelot
+, uharfbuzz
+}:
+
+buildPythonPackage rec {
+  pname = "fpdf2";
+  version = "2.7.6";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "py-pdf";
+    repo = "fpdf2";
+    rev = version;
+    hash = "sha256-wiCKmS+GlrYV2/6TEdXUbmWIMWU4hyzswFJZR9EOWxc=";
+  };
+
+  postPatch = ''
+    substituteInPlace setup.cfg \
+      --replace "--cov=fpdf --cov-report=xml" ""
+  '';
```

```suggestion
  '';
  
  nativeBuildInputs = [
    setuptools
  ];
```
It looks like some of the dependencies propagate setuptools, but we need to add it to nativeBuildInputs as well explicitly.

Besides this, LGTM.